css/homestyle.css

```diff
@@ -0,0 +1,123 @@
+body, html {
+    padding: 0;
+    margin: 0;
+    height: 100%;
+}
+.coverImage {
+    background-color: #000;
+    background: url("../Photos/hero-image 2.png") no-repeat center center fixed; 
+    background-size: cover;
+    position: relative;
+    width: 100%;
+    height: 100%;
+}
+.coverContainer {
+    width: 100%;
+    height: 100%;
+}
+.sidebar {
+    background-color:#FFFFFF;
+    top:0;
+    bottom:0;
+    position: absolute;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+.text {
+    font-family: Sofia Pro;
```

Comment:
aici practic e doar font, daca adaugai si alte font-uri in afara de Sofia Pro, practic era font-family. 
ex. font-family: Sofia Pro, Helvetica, Times New Roman etc.